docs/performance.md

```diff
@@ -55,54 +55,36 @@ It is recommended to use the plugin even if no `console.*` calls are made in you
 
 ### `ListView` initial rendering is too slow or scroll performance is bad for large lists
```

Comment:
Since this component isn't even in core anymore afaik, maybe this section should be removed?

Reply:
I agree.